plugin/pkg/scheduler/core/scheduling_queue.go

```diff
@@ -175,6 +178,7 @@ func NewPriorityQueue() *PriorityQueue {
 	pq := &PriorityQueue{
 		activeQ:        newHeap(cache.MetaNamespaceKeyFunc, util.HigherPriorityPod),
 		unschedulableQ: newUnschedulablePodsMap(),
+		podBackoff:     util.CreateDefaultPodBackoff(),
```

Comment:
I think we should create the backoff with different args. I would suggest:
`CreatePodBackoff(200 * time.Millisecond, 30 * time.Second)`